whisper/timing.py

```diff
@@ -215,6 +215,8 @@ def find_alignment(
 
     words, word_tokens = tokenizer.split_to_word_tokens(text_tokens + [tokenizer.eot])
     word_boundaries = np.pad(np.cumsum([len(t) for t in word_tokens[:-1]]), (1, 0))
+    if len(word_boundaries) <= 1:
```

Comment:
This fixes crashes because word_boundaries could be empty.

```
>>> word_tokens = [[5, 1],[3,2,1], [1]]
>>> np.pad(np.cumsum([len(t) for t in word_tokens[:-1]]), (1, 0))
array([0, 2, 5])
>>> word_tokens = []
>>> np.pad(np.cumsum([len(t) for t in word_tokens[:-1]]), (1, 0))
array([0.])
```